<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewCriteriaPropertiesWidget</class>
 <widget class="QWidget" name="ViewCriteriaPropertiesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>369</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QGroupBox" name="boxSelectByStatus">
       <property name="title">
        <string>Select items by status</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QListWidget" name="listSelectByStatus">
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <item>
           <property name="text">
            <string>with &quot;Good&quot; status</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&quot;Bad&quot; items (new)</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&quot;Bad&quot; items (acknowleged)</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&quot;Unknown&quot; (new)</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&quot;Unknown&quot; (acknowleged)</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>&quot;Wait for master&quot;</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Out of schedule</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Not tested yet</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Disabled</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Warning items (new)</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Warnings items (acknowleged)</string>
           </property>
           <property name="checkState">
            <enum>Unchecked</enum>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="boxSelectByMethod">
       <property name="title">
        <string>Select items by test method</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QListWidget" name="listSelectByMethod">
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="boxSelectByStats">
       <property name="title">
        <string>Select items by stats</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbAliveRatio">
            <property name="text">
             <string>Alive ratio &gt;</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinAlive">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>99.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rbDeadRatio">
            <property name="text">
             <string>Dead ratio &gt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinDead">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>99.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QRadioButton" name="rbUnknownRatio">
            <property name="text">
             <string>Unknown% &gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="spinUnknown">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>99.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="horizontalSpacing">
           <number>0</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbReplyGreater">
            <property name="text">
             <string>Reply &gt;           </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="spinReplyGreater">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>99.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rbReplyLess">
            <property name="text">
             <string>Reply &lt;</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="spinReplyLess">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>99.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="rbStatusDuration">
          <property name="text">
           <string>Status duration</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="cmbDurationCriteria">
            <item>
             <property name="text">
              <string>&gt;</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>&lt;</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinDurationValue">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblMin">
            <property name="text">
             <string>min</string>
            </property>
            <property name="buddy">
             <cstring>spinDurationValue</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="boxSelectByProperties">
     <property name="title">
      <string>Select items by test properties</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0">
         <widget class="QCheckBox" name="chkTestName">
          <property name="text">
           <string>Test name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbTestName">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="chkComment">
          <property name="text">
           <string>Comment</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbComment">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="chkTarget">
          <property name="text">
           <string>Target</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cmbTarget">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="chkTestBy">
          <property name="text">
           <string>Test by/agent</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbTestBy">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="boxSelectUsingExpression">
     <property name="title">
      <string>Select items using expression</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>7</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>7</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="cmbSelectUsingExpression">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::InsertAtTop</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnSelectExpression">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../qHostMonitor.qrc">
          <normaloff>:/img/loupe.png</normaloff>:/img/loupe.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
